health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/CreateChecklist.js

```
@@ -319,8 +361,18 @@ module = "HCM";
       const data = await mutateAsync(payload); // Use mutateAsync for await support
       // Handle successful checklist creation  
       // Proceed with localization if needed
+      console.log("data success", data);
+      console.log("ul", uniqueLocal);
+      let checklistTypeTemp = checklistType.toUpperCase().replace(/ /g, "_");
+      if(checklistTypeCode) checklistTypeTemp=checklistTypeCode;
+      let roleTemp = role.toUpperCase().replace(/ /g, "_");
```

Biome

[error] This let declares a variable that is only assigned once.

'roleTemp' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)